File: example/src/main/java/ru/virgil/example/image/PrivateImageFileDto.java
package ru.virgil.example.image;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import ru.virgil.example.system.IdentifiedDto;


@EqualsAndHashCode(callSuper = true)
@Data
@NoArgsConstructor
public class PrivateImageFileDto extends IdentifiedDto {

    private String name;

}

File: example/src/main/java/ru/virgil/example/image/PrivateImageFileDto.kt
package ru.virgil.example.image

import ru.virgil.example.system.dto.IdentifiedDto
import java.time.LocalDateTime
import java.util.*

data class PrivateImageFileDto(
    override var uuid: UUID?,
    override var createdAt: LocalDateTime?,
    override var updatedAt: LocalDateTime?,
    var name: String?,
) : IdentifiedDto
